### PR TITLE
Update XzBinaryLocator.php

### PR DESCRIPTION
**Pull request by SanoKriss**
Added Windows host support

### CODE REVIEW

File: src/Xz/XzBinaryLocator.php [hunk 1]
```diff
@@ -19,7 +19,13 @@ public function __construct(?string $path)
     public function getXzBinary(): string
     {
         if ($this->path === null) {
-            exec('which xz', $output, $return);
+            if (stristr(PHP_OS, 'LINUX')) {
+                exec( 'which xz', $output, $return );
+            } elseif (stristr(PHP_OS, 'WIN')) {
+                exec( 'where xz', $output, $return );
+            } else {
+                throw new QrPaymentException("not supported OS");
```
Comment (RikudouSage) [inline src/Xz/XzBinaryLocator.php:27]: I would like more descriptive exception message. Including mention of the setXzBinary() method to help guide people on how to solve the error.

**Review by RikudouSage — CHANGES_REQUESTED** [review]
I'm missing some tests for the behavior.

File: src/Xz/XzBinaryLocator.php (hunk 1 above)
Comment (SanoKriss) [reply src/Xz/XzBinaryLocator.php]: added

**Review by SanoKriss — COMMENTED** [review]
added comment

**Pull request closed without merging**